app/modules/users/permissions/rules.py

```diff
@@ -401,6 +402,13 @@ def check(self):
             )
         return has_permission
 
+    def _permitted_as_public_data(self):
+        # All public data can be read/edited by researchers and data managers as decreed
+        # https://wildme.atlassian.net/browse/DEX-1281
+        return self._obj.is_public() and (
```

Comment:
i am looking at how `is_public()` is defined, and it appears to chain off of `user_is_owner(public-user)`.  notably, `indiv.user_is_owner(user)` only requires that the `user` owns **one single** encounter to return `True`.

i am trying to figure out if this is _overly permitting_ for the ticket, in the case of Individuals.  but i have to admit, i always feel a little fuzzy when it comes to _Individual permissions_.   this would mean that a **single** encounter owned by public-user would cause the Individual to be writable by any researcher.  does this change the behavior/permissions around merging or anything else?   i am going to stare at the roles spreadsheet and try to think about that a bit.